cronjob-scale/cronjob.yaml

```diff
@@ -28,6 +28,7 @@ spec:
               command:
                 - scale.sh
                 - "10"
+                - "true"
```

Comment:
Try to use a named flag to avoid violation of SRP (single responsible principle). `--multiply` maybe?

We can try to separate scale up and down into two scripts, defining them on `ConfigMap`, but I think the example will be more complex to understand...